platforms/core-runtime/base-services/src/main/java/org/gradle/internal/service/DefaultServiceRegistry.java

```diff
@@ -108,7 +109,7 @@ public DefaultServiceRegistry(ServiceRegistry... parents) {
         this(null, parents);
     }
 
-    public DefaultServiceRegistry(String displayName, ServiceRegistry... parents) {
+    public DefaultServiceRegistry(@Nullable String displayName, ServiceRegistry... parents) {
```

Comment:
🤔 It seems the field should be annotated with `@Nullable` as well, right?

Reply:
It might, but so do many other fields that get assigned from now-annotated parameters.
I don't want to further increase the scope of the PR. Especially given that annotating fields gives even less benefits than annotating methods at the moment.

Reply:
Well, you're already touching that exact parameter, it would make a lot of sense to update the field. A missed opportunity otherwise.